testing.rst

```diff
@@ -155,7 +155,11 @@ As an example, a test could look like this::
             );
         }
     }
-
+    
+.. tip::
+    
+    Do not forget to require the ``browser-kit`` component via ``composer require browser-kit``, if you want to use the ``WebTestCase`` class.
```

Comment:
I would start the section (i.e. right under the "Your First Functional Test" headline) like this:

```rst
First, install the BrowserKit component in your project:

.. code-block:: terminal

    $ composer require --dev browser-kit
```